frontend/src/components/ModelBox/ModelConfigurationSlider.tsx

```diff
@@ -29,15 +29,13 @@ function ModelConfigurationSlider({
 		setValue(config.value);
 	}, [config]);
 
-	const supportText = `more info about ${config.name}`;
-
 	return (
-		<fieldset>
+		<fieldset className="model-config-slider-fieldset">
 			<legend>{config.name}</legend>
 			<button
 				className="info-icon prompt-injection-min-button"
-				title={supportText}
-				aria-label={supportText}
+				title="more info"
+				aria-label="more info"
```

Comment:
just a tiny preference: when I was screen reading, having "info" read out by a robot felt kinda weird, but that's just my opinion. I'd prefer "information", but I guess this is cool if it's more consistent with the tone of the rest of the app.

Reply:
saves 2 vowels of time though!

Reply:
Didn't consider that!